packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/FormRow/formParts/SelectiveBreeding/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { compose } from 'redux'
+import { Grid } from 'semantic-ui-react'
+import { ModuleTranslate } from 'coreModules/i18n/components'
+import { Radio } from 'coreModules/form/components'
+import { withI18n } from 'coreModules/i18n/higherOrderComponents'
+import {
+  formatBooleanRadio,
+  parseBooleanRadio,
+} from 'coreModules/form/utilities'
+
+import createLog from 'utilities/log'
+
+const log = createLog(
+  'modules:collectionMammals:MammalForm:OriginInformation:SelectiveBreeding'
```

Comment:
this log should be more like the folder path e.g. not MammalForm:OriginInformation but MammalManager:MainColumn etc (but I know this is one thing that we need to clean up later)